mlir/test/mlir-miopen-driver/populate_host_print.mlir

```diff
@@ -0,0 +1,119 @@
+// RUN: mlir-miopen-driver -p -ph -pr | FileCheck %s --check-prefix=F32
+// RUN: mlir-miopen-driver -p -ph -pr -t f16 | FileCheck %s --check-prefix=F16
+// RUN: mlir-miopen-driver -p -ph -pr -t bf16 | FileCheck %s --check-prefix=BF16
+
+// F32-LABEL: func @main()
+// F32-NEXT: alloc() : memref<128x8x3x3xf32>
+// F32-NEXT: alloc() : memref<128x8x32x32xf32>
+// F32-NEXT: alloc() : memref<128x128x30x30xf32>
+// F32-NEXT: alloc() : memref<128x128x30x30xf32>
```

Comment:
Same here: Use string substitution for different data types.

Reply:
follow-up PR in #69.